### PR TITLE
Use the main GitHub README.md file for NPM

### PR DESCRIPTION
**Pull request by dxbykov**

### CODE REVIEW

**Review by BingoRUS — COMMENTED** [review]
This decision looks quite strange. We're going to talk about NuGet and Windows installer on npmjs.com. There is only jQuery example which looks weird for guys using npm. There is no info about bundler.

**Comment by AlekseyMartynov** [comment]
The current repo's README indeed doesn't really fit.
For example:

> This is a working repo for DevExtreme. To include...

@BingoRUS 
If I remember correctly, the links to specific sections (NuGet, Windows installer, etc) are there because there's currently no way to link the entire 'Installation' section. If we solve this, then we can squeeze 'Download and Install' and 'Getting Started' parts, and use the single README.

**Comment by dxbykov** [comment]
@BingoRUS  I believe GitHub and NPM are entry points for the same audience. If it's not clear how to get started from our Readme then let's improve it.

File: README.md [hunk 1]
```diff
@@ -29,9 +29,10 @@ After installation, you can add a widget to your app. Here's a simple [button ex
 ```
 
 ```js
-$(function () {
-    $("#buttonContainer").dxButton();
-});
+import Button from 'devextreme/ui/button';
+
+let element = document.getElementById('buttonContainer');
+let button = new Button(element, { text: 'Hello World!' });
```
Comment (San4es) [inline README.md]: Seems, we can simplify to:
let button = new Button("#buttonContainer", { text: 'Hello World!' });

Comment (dxbykov) [reply README.md]: This way, I have no idea if it's based on jQuery or not. I would keep the most transparent example when the Button accepts a pure JS element.

Comment (AlekseyMartynov) [reply README.md]: `const` 😄

Comment (San4es) [reply README.md]: `new Button("#buttonContainer")`
It will work with jQuery and without it as well.

**Review by AlekseyMartynov — DISMISSED** [review]
🆗

**Review by AlekseyMartynov — APPROVED** [review]

**Pull request merged**